lib/spack/spack/detection/path.py

```diff
@@ -5,6 +5,7 @@
 and running executables.
 """
 import collections
+import collections.abc
```

Comment:
:thinking: I would have expected this to be flagged since we don't use `collections.abc` anymore, right?

Reply:
It has side effects from code running in global scope, so it can't technically be marked as redundant.